client/actions.js

```diff
@@ -1,14 +1,41 @@
+const MESSAGE_RECEIVED = 'MESSAGE_RECEIVED'
+const messageReceived = text => ({ type: MESSAGE_RECEIVED, text })
```

Comment:
I don't think `MESSAGE_RECEIVED` or `MESSAGE_CLEARED` are needed yet. You can double-check me.